### DISCUSSION TITLE
[Feature Request] GrapeJS field type

### DISCUSSION THREAD

**Discussion by tabacitu** [Laravel-Backpack/community-forum#311] [General]
Page builder field using GrapeJS... maybe?
https://grapesjs.com/

**Comment by dauvray** [comment]
Gutemberg via laraberg is a good choice for a custom field

**Comment by tabacitu** [comment]
Yup - there's an open issue for that too 😄 https://github.com/Laravel-Backpack/CRUD/issues/1872

**Comment by dauvray** [comment]
Maybe grapesjs is good for the pagemanager package in addition to the template system. I think they don't have the same goal.

**Comment by bedoz** [comment]
> Page builder field using GrapeJS... maybe?
> https://grapesjs.com/

I think it's a very cool idea, it should be use for any drag'n'drop style pages.
